edge-agent/docker/linux/arm32v7/Dockerfile

```diff
@@ -1,23 +1,22 @@
 # syntax=docker/dockerfile:1.4
 
-FROM mcr.microsoft.com/dotnet/aspnet:6.0-focal-arm32v7
+FROM mcr.microsoft.com/dotnet/aspnet:6.0-alpine
 
-RUN apt-get update && \
-    apt-get install -y libsnappy1v5 && \
-    ln -s /lib/arm-linux-gnueabihf/libdl.so.2 /usr/lib/arm-linux-gnueabihf/libdl.so && \
-    ln -s /lib/arm-linux-gnueabihf/libc.so.6 /usr/lib/arm-linux-gnueabihf/libc.so && \
-    rm -rf /var/lib/apt/lists/*
```

Comment:
Do you know why we needed these symlinks?

Reply:
No. :) I assume it was because the dependent programs were looking for the shared libraries by their version-independent names.